Hide password params.

djwhatle opened the pull request:
- When collecting params with `DisplayType: password`, turn off echo.
 - Clean up selectParameters function

djwhatle commented:
Addresses https://github.com/automationbroker/sbcli/issues/38

dymurray commented on pkg/runner/runner.go:
```diff
@@ -176,15 +178,28 @@ func selectParameters(plan bundle.Plan) (bundle.Parameters, error) {
-		var check = true
-		for check {
+
+		for !inputValid {
```
+1 much better :)

jmrodri replied:
This is better.

dymurray commented:
Just need to rebase.

jmrodri commented on pkg/runner/runner.go:
```diff
@@ -176,15 +178,28 @@ func selectParameters(plan bundle.Plan) (bundle.Parameters, error) {
 	schemaParams := planSchema.ServiceInstance.Create["parameters"]
 	params := bundle.Parameters{}
 	for _, param := range plan.Parameters {
+		var inputValid = false
```
`var inputValid bool` would be better since the zero value for booleans is false. This would be inline like the line below where we do `var paramDefault interface{}`.

djwhatle replied:
From a code readability perspective, it feels clearer to me to explicitly set `inputValid` to false here, but perhaps this is a Golang convention I'm not privy to quite yet :)

djwhatle commented:
@dymurray rebased and updated `Gopkg.toml` and `Gopkg.lock`. It turns out we already have this library in our vendor dir, maybe as a dependency of one of the other libraries. Added to `Gopkg.*` anyways so that if we remove those libs later for some reason, we should still have this.
